Returns null when there is no matching block (#410)
klaytn/klaytn-docs: docs/dapp/json-rpc/api-references/eth/block.md

**Return Value**

> `Object` - A header object, or `null` when no header was found. It returns error when failed to fetch block proposer(= miner) address:

Reviewer: ```suggestion
`Object` - A header object, or `null` when no header was found. Otherwise, it returns an error.
```

I think this is enough To cover the other error cases in the future.

Author: @kjhman21 
Got it. I'll update this PR :)

Author: PTAL [this commit](https://github.com/klaytn/klaytn-docs/pull/410/commits/60e0593af3d27a042bb28bd6db593aa89bc25672)

**Return Value**

> `Object` - A block object, or `null` when no block was found:

Author: Sorry for it. 
There is a case when error can occur. (PTAL [this comment](https://github.com/klaytn/klaytn/pull/1261#discussion_r832731393))
I'll update this PR soon.
